homeassistant/components/husqvarna_automower/strings.json

```
@@ -248,6 +248,9 @@
     "switch": {
       "enable_schedule": {
         "name": "Enable schedule"
+      },
+      "stay_out_zones": {
+        "name": "Enable {stay_out_zone}"
```

I have a question about what enabled means.
From the code, enabled means the mower would avoid the zone, but from the translation, it could also mean the opposite.

Example: 
Zone named `Kids zone`
If I now have a switch named `Enabled Kids zone`, I would assume that when the switch is activated, the mower also cuts the kids zone.

Maybe we should call it:
```suggestion
        "name": "Avoid {stay_out_zone}"
```